src/kw_db.sh

```diff
@@ -28,3 +28,26 @@ function execute_sql_script()
 
   sqlite3 "$db_path" < "$sql_path"
 }
+
+# This function reads and executes a SQL script in the database
+#
+# @sql_cmd:   SQL command to be executed on @db
+# @db:        Name of the database file
+# @db_folder: Path to the folder that contains @db
+#
+# Return:
+# 2 if db doesn't exist;
+# 0 if succesful; non-zero otherwise
+function execute_command_db()
+{
+  local sql_cmd="$1"
+  local db="${2:-"$DB_NAME"}"
+  local db_folder="${3:-"$KW_DATA_DIR"}"
+  local db_path
+
+  db_path="$(join_path "$db_folder" "$db")"
+
+  file_exists "$db_path" || exit_msg 'Database does not exist' 2
+
+  sqlite3 "$db" -batch "$sql_cmd"
```

Comment:
I did some tests, and `sqlite3` prints a message to the terminal detailing the problem with the command. I could add the `-bail` flag, which ensures sqlite3 stops as soon as there is an error.
I think we could add a message that displays the command that caused the error, as `sqlite3` only displays the problematic part.

Reply:
We should return something in case of a problem and let the function that invokes it handle the issue. However, I don't think we should add any warning message here because this is not something the user will understand in case of an error. Ideally, we should have the verbose or debug mode for handling these cases. (https://github.com/kworkflow/kworkflow/issues/179)